css/css-shapes/shape-outside/shape-image/gradients/shape-outside-radial-gradient-001.html

```diff
@@ -55,6 +55,7 @@
     </div>
     <div id="log"></div>
     <script>
+        setup({ single_test: true });
```

Comment:
`verifyTextPoints` is synchronous, can you remove the `done()` call from it and just wrap this in `test(() => {})` instead?

Reply:
Sure. As long as we're refactoring (and in keeping with the spirit of your suggestion), I've also moved the assertion into the test body.

Reply:
I think it would have been fine to leave it in the helper and that would have made the call sites look a bit simpler, but sure, there is a consistency to extracting that too.